qiskit-ibm-runtime-mcp-server/src/qiskit_ibm_runtime_mcp_server/ibm_runtime.py

```diff
@@ -753,4 +753,130 @@ async def get_service_status() -> str:
         return f"IBM Quantum Service Status: {status_info}"
 
 
+@with_sync
+async def delete_saved_account(account_name: str = "") -> str:
+    """Delete a saved account.
+    Args:
+        account_name: Name of the saved account to delete.
+
+    Returns:
+    Information about if an account was deleted.
+    """
+    global service
+
+    try:
+        if service == "":
+            service = initialize_service()
+        is_deleted = service.delete_account(name=account_name)
+
+        if is_deleted:
+            delete_info = {"deleted": True}
+            return f"Account successfully deleted: {delete_info}"
+        else:
+            return f"Failed to delete the account {account_name}, please validate the account name exists."
+
+    except Exception as e:
+        logger.error(f"Failed to delete account: {e}")
+        delete_info = {"deleted": False, "error": str(e)}
+        return f"Error deleting account: {delete_info}"
+
+
+@with_sync
+async def list_saved_account() -> str:
```

Comment:
The existing codebase returns `dict[str, Any]` for most operations, but all new functions return `str`. 

This inconsistency will affect:

- Type safety and IDE autocomplete
- Client code that expects structured data
- Ability to programmatically parse responses

Recommendation: Return `dict[str, Any]` with structured fields like `{"status": "success", "accounts": [...]}` instead of formatted strings like `f"Account list: {accounts_list}".`